src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -15,16 +15,45 @@ Add `AddAWSLambdaConfigurations()` to `TracerProvider`.
 ```csharp
 TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
         // add other instrumentations
-        .AddAWSLambdaConfigurations()
+        .AddAWSLambdaConfigurations(options => options.DisableAwsXRayContextExtraction = true)
         .Build();
 ```
 
+### AWSLambdaInstrumentationOptions
+
+`AWSLambdaInstrumentationOptions` contains various properties to configure
+AWS lambda instrumentation.
+
+* [`DisableAwsXRayContextExtraction`](/src/OpenTelemetry.Instrumentation.AWSLambda/AWSLambdaInstrumentationOptions.cs)
+* [`SetParentFromBatch`](/src/OpenTelemetry.Instrumentation.AWSLambda/AWSLambdaInstrumentationOptions.cs)
+
 ## Instrumentation
 
+`AWSLambdaWrapper` class contains tracing methods covering different types of
+function handler method signatures. `AWSLambdaWrapper.Trace()` and
+`AWSLambdaWrapper.TraceAsync()` methods are used for wrapping synchronous
+and asynchronous function handlers respectively. The `ActivityContext parentContext`
+parameter is optional and used to pass a custom parent context. If the parent
+is not passed explicitly then it's either extracted from the
+input parameter or uses AWS X-Ray headers if AWS X-Ray context extraction is
+enabled (see configuration property `DisableAwsXRayContextExtraction`).
+The sequence of the parent extraction:
+`explicit parent` -> `parent from input parameter` -> `AWS X-Ray headers` -> `default`
+The parent extraction is supported for the input types listed in the table below.
+Although the `SQSEvent` and `SNSEvent` consist of multiple
+records (`SQSMessage` and `SNSRecord`) it's recommended to pass the full event.
+
+| Package | Types |
```

Comment:
I wonder if there is any  value in listing the Package? Instead, it would be good to mention briefly where it is extracted from (e.g. for SQSEvent: Message attributes of last message; API Gateway events: HTTP headers)

Reply:
I replaced the 'Package' column by the 'Parent extraction source'.